### DISCUSSION TITLE
Godot bindings for Swift

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [godotengine/godot-proposals#6920] [Scripting]
[Godot bindings for Swift](https://github.com/migueldeicaza/SwiftGodot)

Incredible, @migueldeicaza and his contributors did it in short time

https://github.com/godotengine/godot-proposals/assets/49812372/926a8b5e-80ca-4d19-8156-4d7015eeb24b

**Comment by migueldeicaza** [comment]
:-)
